up-l1/README.adoc

```diff
@@ -202,11 +171,15 @@ unregisterListener(
 | The listener to be unregistered.
 
 | result
-| Future<Void, link:../basics/error_model.adoc[UStatus]>
+| Future<Void>
 | A succeeded future if the listener has been unregistered successfully. +
-Otherwise, a failed future containing a `UStatus` indicating the reason for the failure.
+Otherwise, a failed future which contains a `UCode` indicating the reason for the failure.
 |===
 
+Implementations
+
+* *MAY* fail invocations with a `UCode::UNIMPLEMENTED` if the transport does not support the _push_ <<_delivery_method>>. In that case, the <<_registerlistener>> method *MUST* also fail accordingly.
```

Comment:
Same thing appears to be happening as [here](https://github.com/eclipse-uprotocol/up-spec/pull/134/commits/03d54831bc214ee82e6fe2ee2895ed387a8ef02a#r1602292080).